src/Native/Unix/System.Native/pal_interfaceaddresses.c

```diff
@@ -38,6 +44,42 @@
 #include <net/route.h>
 #endif
 
+static inline uint8_t mask2prefix(uint8_t * mask, int length)
```

Comment:
Can you add a comment explaining what this function is doing?  It's not clear to me, and in particular what the mask is and what the prefix is it's converting to (since it looks like it's returning a length).